activestorage/test/models/attachments_test.rb

```diff
@@ -68,7 +68,7 @@ class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
   test "attach new blobs" do
     @user.highlights.attach(
       { io: StringIO.new("STUFF"), filename: "town.jpg", content_type: "image/jpg" },
-      { io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg" })
+      io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg")
```

Comment:
Yeah, the changes in `select` and `form_tag` really annoys me. But since it is only on tests I'm fine with it.